src/composables/queries/usePoolGaugesQuery.ts

```diff
@@ -38,10 +38,12 @@ export default function usePoolGaugesQuery(
   /**
    * COMPUTED
    */
+  const enabled = computed((): boolean => !!poolAddress?.value);
+
   const subgraphQuery = computed(() => ({
     pool: {
       __args: {
-        id: poolAddress.value.toLowerCase(),
+        id: poolAddress?.value?.toLowerCase(),
```

Comment:
it seems we don't expect undefined in `id` field, so it possibly better to put above check for poolAddress.value

```
if (!poolAddress.value) {
   return {}
}
```
btw, there is no need in optional chaining after poolAddress, due to its type `Ref<string | undefined>`, may be you mean `Ref<string | undefined> | undefined`?